### PR TITLE
chore: updating create nft collection

### PR DESCRIPTION
**Pull request by NMINHDAI**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: pkg/handler/nfts.go [hunk 1]
```diff
@@ -30,6 +35,19 @@ func (h *Handler) CreateNFTCollection(c *gin.Context) {
 	}
 	checksumAddress, _ := util.ConvertToChecksumAddr(req.Address)
 
+	checkExitsNFT, err := h.entities.CheckExistNftCollection(checksumAddress)
+	if err != nil {
```
Comment (trkhoi) [inline pkg/handler/nfts.go:39]:
```
if err != nil {
	c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
	return
}
```

**Review by trkhoi — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]
